src/rustllvm/PassWrapper.cpp

```diff
@@ -425,6 +425,9 @@ extern "C" void LLVMRustPrintTargetFeatures(LLVMTargetMachineRef TM) {
   for (auto &Feature : FeatTable)
     printf("    %-*s - %s.\n", MaxFeatLen, Feature.Key, Feature.Desc);
   printf("\n");
+  // Rust specific target features
```

Comment:
If we're going to have this block separated by an empty line, perhaps it should print a new header too, like "Rust-specific features:".

Reply:
thanks, makes sense. done